pacemaker.spec.in

```diff
@@ -26,8 +26,18 @@
 %global py_site    %{?python_sitearch}%{!?python_sitearch:%(python -c "from distutils.sysconfig import get_python_lib; print(get_python_lib(1))")}
 %global cman_allow (0%{?fedora} > 0 && 0%{?fedora} < 17) || (0%{?rhel} > 0 && 0%{?rhel} < 7)
 
-# https://fedoraproject.org/wiki/EPEL:Packaging?rd=Packaging:EPEL#The_.25license_tag
-%{!?_licensedir:%global license %doc}
+# It's nice to use the license macro uniformly throughout the spec below,
+# but RPM versions not aware of this new files' tagging normally (re)define
+# such macro to the value of "License:"; fortunately, Jason Tibbitts came
+# with an elegant solution that injects expected license value after
+# "License:", making it effectively used in files section correctly:
+# https://pkgs.fedoraproject.org/cgit/rpms/epel-rpm-macros.git/tree/macros.zzz-epel?h=el6&id=e1adcb77b0c05a6c29bc0eb0c4e82113180a0a99#n12
+%if !%{defined _licensedir}
+%define description %{lua:
+    rpm.define("license %doc")
+    print("%description")
+}
+%endif
```

Comment:
I made an extra check and found that SuSE has lua scripts support enabled since 2006.
And it is also a default for RPM build (unless `--without-lua`). So I don't sense any controversies, also given the prior statement.